pkg/operator/ceph/disruption/clusterdisruption/pools.go

```diff
@@ -201,7 +201,10 @@ func (r *ReconcileClusterDisruption) reconcileCephFilesystem(cephFilesystemList
 
 		activeCount := filesystem.Spec.MetadataServer.ActiveCount
 		minAvailable := &intstr.IntOrString{IntVal: activeCount - 1}
-		if minAvailable.IntVal <= 1 {
+		if filesystem.Spec.MetadataServer.ActiveStandby {
+			minAvailable.IntVal++
+		}
+		if minAvailable.IntVal < 1 {
```

Comment:
Why not `<=` just like previously?

Reply:
It was inaccurate before. Should I mention this in the commit message?

Reply:
I'll do that. I forgot :P

Reply:
I think the commits and description are fixed to explain this now.